web/src/components/alerting/AlertsDetailPage.tsx

```diff
@@ -128,20 +120,6 @@ const AlertsDetailsPage_: React.FC<AlertsDetailsPageProps> = ({ history, match }
 
   const AlertsChart = alertsChart?.[0];
 
-  const [customExtensions] =
```

Comment:
Was this custom extension alert source, moved elsewhere. This allows log based alerts to have a detail page.